modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/components/containers/Forms/Basics/BusinessType/RadioGroup.js

```diff
@@ -30,6 +30,10 @@ export const BusinessTypeRadioGroup = ({businessTypes = []}) => {
 				'basics.properties.businessClassCode',
 				categoryProperties.find(({key}) => key === 'BCC').value
```

Comment:
The same from below, BCC may not exist for some business

`?.value`

Reply:
done!

Reply:
Thank you, man!!